### ISSUE TITLE
Build failure in devel: class "workFlow" is not exported by 'namespace:flowCore'

### ISSUE THREAD

**Issue by sneumann** [jspidlen/flowUtils#3]
Hi,
flowUtils fails to build in 
http://bioconductor.org/checkResults/devel/bioc-LATEST/flowUtils/malbec1-install.html
due to 
```
Error: class "workFlow" is not exported by 'namespace:flowCore'
```
btw, a similar issue occurs in 
https://github.com/RGLab/CytoML/issues/54

Apparently there was an API change removing `workFlow` in 
https://github.com/RGLab/flowCore/commit/24d80163c638a0d9b027379b8db4d5b5d8f876f6#diff-7347fe5a0f184f79ef064e92e3beb297L167

Yours, 
Steffen

**Comment by jacobpwagner** [comment]
@sneumann, from my comment on the parallel issue in `CytoML`: https://github.com/RGLab/CytoML/issues/54

My PR to remove the dependency was merged here weeks ago: jspidlen/flowUtils@e66653c.

However, that commit has to be merged to Bioconductor by @jspidlen. I have notified him of the issue, but until it is fixed through Bioconductor, users can install flowUtils from the GitHub repo, where the dependency is removed.

**Comment by jspidlen** [comment]
I am aware of this but still don't know how to merge from
https://github.com/jspidlen/flowUtils to whatever the BioConductor version
is; can someone send me some instructions? And I apologize for my ignorance
on this, haven't done anything here in years ...

Best,
Josef

**Comment by jacobpwagner** [comment]
No worries, @jspidlen. I sent instructions to your bd email on Friday I think. Basically you just need to merge the GitHub `master` branch here to the `master` branch of git@git.bioconductor.org:packages/flowUtils.git

If you increase the version in the `DESCRIPTION` file, it should make sure the change gets picked up for upcoming builds.

Alternatively, I can embed the necessary `flowUtils` code in to `CytoML` and remove `CytoML`'s dependency on `flowUtils`, but `flowUtils` will continue to fail because `flowCore::workFlow` is removed.

**Comment by jspidlen** [comment]
Right, now I remember... I tried to do something about it couple weeks ago,
ended up not having access to git@git.bioconductor.org:packages/flowUtils.git,
so tried to use
https://git.bioconductor.org/BiocCredentials/account_activation/ to
activate my account, but it told me that my email is not associated with
any Bioconductor packages although it should be. So I just gave up. I'll
try to see if bioc-devel@r-project.org can help :-(.
